core/lib/types/src/commitment/tests/post_boojum_1_4_2_test.json

```diff
@@ -485,7 +485,8 @@
   "meta_parameters": {
     "zkporter_is_available": false,
     "bootloader_code_hash": "0x010007ed0e328b940e241f7666a6303b7ffd4e3fd7e8c154d6e7556befe6cd6d",
-    "default_aa_code_hash": "0x0100055b7a8be90522251be8be1a186464d056462973502ac8a0437c85e4d2a9"
+    "default_aa_code_hash": "0x0100055b7a8be90522251be8be1a186464d056462973502ac8a0437c85e4d2a9",
+    "protocol_version": "Version21"
```

Comment:
It would be good to have test for 1.5.0 commitment. You can write a test and change impl of `is_pre_1_5_0`, `is_post_1_5_0` for `#[cfg(test)]`.

Reply:
It was harder to do than anticipated. I talked to @perekopskiy in private, it was decided to include the test into the main 1-5-0-support PR